lib/src/storybook.dart

```diff
@@ -16,41 +16,6 @@ import 'package:storybook_toolkit/src/plugins/text_sizer.dart';
 import 'package:storybook_toolkit/src/plugins/theme/code_view_syntax_theme.dart';
 import 'package:storybook_toolkit/storybook_toolkit.dart';
 
-class _ThemeWrapperBuilder {
-  _ThemeWrapperBuilder();
-
-  static Widget themeBuilder(
```

Comment:
Why did you remove this themeBuilder?

Reply:
No longer used.

Reply:
Yes, I understand. I will merge it and refactor the build method of _StorybookState when I have time. But if you have some ideas and time for contribution, you can make another PR ;)